cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceMQ.java

```diff
@@ -73,7 +73,7 @@ public void sendFinalGrades(int course_id, FinalGradeDTO[] grades) {
 
 		//TODO convert grades to JSON string and send to registration service
 		String jsonGrade = asJsonString(grades);
-		rabbitTemplate.convertAndSend("registration-exchange", "registration-routing-key", jsonGrade);
+		rabbitTemplate.convertAndSend(registrationQueue.getName(), jsonGrade);
```

Comment:
This update correctly makes it so the message is send. The registration queue gets the name, which corresponds to the same name in the other service and the message is exchanged